IlmBase/config/CMakeLists.txt

```diff
@@ -76,8 +77,8 @@ if(ILMBASE_INSTALL_PKG_CONFIG)
   function(ilmbase_pkg_config_help pcinfile)
     set(prefix ${CMAKE_INSTALL_PREFIX})
     set(exec_prefix "\${prefix}")
-    set(libdir "\${exec_prefix}/${CMAKE_INSTALL_LIBDIR}")
-    set(includedir "\${prefix}/${CMAKE_INSTALL_INCLUDEDIR}")
+    join_paths(libdir "\${exec_prefix}" "${CMAKE_INSTALL_LIBDIR}")
```

Comment:
is this kind of path joining done only here in the pkg_config setup? I like the idea of join_paths, and am wondering if there are other places to touch, for consistent usage across our scripts?

Reply:
I tried `grep`ing for `CMAKE_INSTALL_` and did not see anywhere else where it is used as anything but leftmost component of path.

Reply:
cool, thanks